src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue30004.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue30004 : _IssuesUITest
+{
+	public override string Issue => "FontImageSource not center-aligned inside Image control";
+
+	public Issue30004(TestDevice device)
+	: base(device)
+	{ }
+
+	[Test]
+	[Category(UITestCategories.Image)]
+	public void VerifyFontImageAreCenterAlign()
+	{
+		App.WaitForElement("FontImage");
+		VerifyScreenshot();
```

Comment:
Pending snapshots.
![image](https://github.com/user-attachments/assets/eeeed661-489d-43a5-af22-5cead76454e7)
Could you commit the images?

Reply:
@jsuarezruiz, I have committed the images.